README.md

```diff
@@ -58,7 +58,7 @@ Add a script tag referencing mapbox-gl-leaflet after adding leaflet in your webs
 ```
 You can also use Unpkg as a CDN using:
 ```html
-<script src="https://unpkg.com/mapbox-gl-leaflet@0.0.3/leaflet-mapbox-gl.js"></script>
+<script src="https://unpkg.com/mapbox-gl-leaflet@0.0.8/leaflet-mapbox-gl.js"></script>
```

Comment:
we're bound to forget to bump that link again. it'd probably be best to just use an evergreen url.
```suggestion
<script src="https://unpkg.com/mapbox-gl-leaflet/leaflet-mapbox-gl.js"></script>
```